defaults.json

```diff
@@ -0,0 +1,37 @@
+{
+    "0.1.0": {
+        "allowRunningInsecureContent": false,
+        "contextIsolation": false,
+        "experimentalFeatures": false,
+        "images": true,
+        "javascript": true,
+        "nativeWindowOpen": false,
+        "nodeIntegration": true,
+        "nodeIntegrationInWorker": false,
+        "offscreen": false,
+        "plugins": false,
+        "sandbox": false,
+        "textAreasAreResizable": true,
+        "webSecurity": true,
+        "webgl": true,
+        "webviewTag": true
+    },
+    "5.0.0": {
+        "allowRunningInsecureContent": false,
+        "contextIsolation": false,
+        "experimentalFeatures": false,
+        "images": true,
+        "javascript": true,
+        "nativeWindowOpen": false,
+        "nodeIntegration": false,
+        "nodeIntegrationInSubFrames": false,
+        "nodeIntegrationInWorker": false,
+        "offscreen": false,
+        "plugins": false,
+        "sandbox": false,
+        "textAreasAreResizable": true,
+        "webSecurity": true,
+        "webgl": true,
+        "webviewTag": false
+    }
+}
```

Comment:
Ideally creating something that dynamically confirms if these security flags are set correctly and working as intended would be preferable, but I can see how using `getWebPreferences()` is a lot easier in our case. I would settle for this defaults file as of now, maybe planning a more comprehensive version for a next release. Things currently missing that could be useful for future checks:
- `enableRemoteModule` (Default should be true)
- `safeDialogs` (Default should be false)
- `navigateOnDragDrop` (Default should be false)
- `spellCheck` (Default should be true)

Reply:
I'll add those manually for now.

As per [this](https://www.electronjs.org/docs/breaking-changes), `enableRemoteModule` will default to `false` from version 10.0.0, so I will also include a dump from the latest beta of 10. I'll also update the remote module check accordingly.  
The other prefs don't seem to have ever had their default changed.

Reply:
Heh. This also revealed a bug that should have been obvious in hindsight. :D 

(Lexicographically) sorting the version keys using `.sort()` of course produces the wrong result as soon as we get into the double digit major versions. I'll correct this to use `semver`'s `compare()` as the correct compare function instead.